test/e2e_node/services/kubelet.go

```diff
@@ -305,6 +302,10 @@ func (e *E2EServices) startKubelet(ctx context.Context, featureGates map[string]
 		cmdArgs = append(cmdArgs, "--image-service-endpoint", framework.TestContext.ImageServiceEndpoint)
```

Comment:
there are things like `--runtime-cgroups` which are being set above. Are we just assuming it will never be configured by mistake on Windows?